### PR TITLE
transformation_mappings

### LINKED ISSUE #7: Actions and reducers for managing Infrastructure Mapping objects
I'm going to include Cluster Mapping, Datastore Mapping and Network Mapping in this issue/task because it will largely all be the same code (probably).

### PR DESCRIPTION
**Pull request by priley86**
closes #7  (redux actions/reducers for transformation_mappings)

* go ahead and move Cards into individual components

### CODE REVIEW

**Comment by priley86** [comment]
* injects transformationMappings into plan wizard (and handles disable/enable button logic)

**Comment by priley86** [comment]
@AparnaKarve @mturley were you able to integrate this branch into Step 1 of Plan Wizard? Would it be OK to go ahead and merge this to close #7  ?

**Comment by AparnaKarve** [comment]
@priley86 In Plan Wizard Step1, there would be no API calls to fetch the list of transformation mappings? And the dropdown there, should always rely on the injected `transformationMappings` array?

**Comment by AparnaKarve** [comment]
@priley86 I was wondering if the GET request to fetch the transformation mappings needs to happen in the Plan Wizard Step1 rather than in the Overview -- in order to prolong the API call until you actually need it.

**Comment by priley86** [comment]
@AparnaKarve per our discussion - we can keep it in Overview (since it is also used to drive enable/disable button).

also, i have changed the base branch here to `migration-poc`. Let me know if this is OK to merge.

File: app/javascript/react/screens/App/Overview/overview.fixtures.js [hunk 1]
```diff
@@ -4,5 +4,31 @@ export const initialState = Immutable({
...
+    transformation_mapping_items: [
+      { source: '/api/clusters/10', destination: '/api/clusters/1' },
+      { source: '/api/clusters/11', destination: '/api/clusters/1' },
+      { source: '/api/date_stores/12', destination: '/api/data_stores/13' },
+      { source: '/api/data_stores/2', destination: '/api/data_stores/13' },
+      { source: '/api/lans/2', destination: '/api/lans/13' },
+      { source: '/api/lans/3', destination: '/api/lans/13' }
+    ]
+  }
+];
```
Comment (AparnaKarve) [inline app/javascript/react/screens/App/Overview/overview.fixtures.js]: I did not see an `id` field. Can we add that?

Comment (priley86) [reply app/javascript/react/screens/App/Overview/overview.fixtures.js]: updated.

Comment (AparnaKarve) [reply app/javascript/react/screens/App/Overview/overview.fixtures.js]: Thanks - That looks good!

**Comment by AparnaKarve** [comment]
Looks good. Merging.

**Pull request merged**